File: missingNumber.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        array=[]
        
        max_1=max(nums)
        print(max_1)

        for i in range(max_1):
            if i in nums:
                array.append(i)
            else :
                return i 
               

File: Missing Number.js
var missingNumber = function(nums) {

    let n=nums.length 
    let i =0 
    while (i<n+1){
        console.log(i)
        if (nums.indexOf(i)==-1){
            return i 
        }

        i=i+1
    }
    
};
